app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -0,0 +1,37 @@
+angular.module('topologyApp', ['kubernetesUI'])
+.config(['$httpProvider', function($httpProvider) {
+    $httpProvider.defaults.headers.common['X-CSRF-Token'] = jQuery('meta[name=csrf-token]').attr('content');
```

Comment:
Hmm, I agree we need this but it feels *wrong* to have to repeat it in every module.. maybe we can do a `miqHttp` service to wrap `$http` or something.. But I guess that's for a separate PR, just thinking out loud here :)